tools/src/main/java/org/apache/rocketmq/tools/command/acl/GetAccessConfigSubCommand.java

```diff
@@ -116,8 +115,6 @@ private void printClusterBaseInfo(
                     field.setAccessible(true);
                     if (field.get(config) != null) {
                         System.out.printf("%-1s %-18s: %s\n", "", field.getName(), field.get(config).toString());
-                    } else {
```

Comment:
I believe leaving it blank makes sense and would not affect its meaning. Do you think it's really necessary to remove it?